docs/production/export-and-import.md

```diff
@@ -63,29 +63,22 @@ export tarball:
 cd /tmp
 tar -xf /path/to/export/file/zulip-export-zcmpxfm6.tar.gz
 cd /home/zulip/deployments/current
-./manage.py import --destroy-rebuild-database '' /tmp/zulip-export-zcmpxfm6
-./manage reactivate_realm -r ''  # Reactivates the organization
```

Comment:
I believe this is needed if and only if you did the ` ./manage deactivate_realm` above.

Reply:
Got it. I'll add it back in here and in Import options below and then merge

Reply:
Got it. I'll add it back in here and in Import options below and then merge